screenshots/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'd rather just have a single package.json.  We can put all of the packages needed for the screenshots in the root `devDependencies`.